morphic.js

```diff
@@ -7519,6 +7519,10 @@ MenuMorph.prototype.addLine = function (width) {
     this.items.push([0, width || 1]);
 };
 
+MenuMorph.prototype.addSubmenu = function (key, submenu) {
+	this.addItem(key, key, null, null, null, null, null, submenu);
```

Comment:
LOL. But also cringe. this for @jmoenig:
 But I'm using this as a plug to eventually have an API that uses named parameters / objects instead of just positional arguments! Making anything that can be null / optional a key would greatly improve readability and learnability of morphic. :)